src/main/java/decodes/routing/RoutingSpecThread.java

```diff
@@ -175,6 +175,7 @@ public void setCurrentStatus(String currentStatus)
 	public RoutingSpecThread()
 	{
 		super();
+		setDaemon(true);
```

Comment:
Arguably debatable if this is correct here but I was having issues with the RoutingScheduler actually shutting down. Especially with newer JVMs